### PR TITLE
Adding Maxsmi

### PR DESCRIPTION
**Pull request by t-kimber**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by t-kimber** [comment]
Dear @jaimergp,
Windows is failing for this recipe (see [this link](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=423362&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=841)), because of Pytorch. However, since this is a noarch package,
1. would it still be possible to merge it, and
2. add the following instructions to the maxsmi documentation:
```
# Linux / MacOS
mamba install maxsmi

# Windows
mamba install maxsmi -c conda-forge -c defaults
``` 

This is what was done for the TeachOpenCADD project and Tensorflow.

Thank you in advance for your help!

File: recipes/maxsmi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  run:
+    - python >=3.7
+    - jupyter
+    - jupyterlab
+    - rdkit >=2018.09.2.0
+    - scikit-learn
+    - requests
+    - tqdm
+    - pytorch
+    - matplotlib
```
Comment (jaimergp) [inline recipes/maxsmi/meta.yaml:30]: You probably want just `matplotlib-base` here to save the `qt` download (you are only using matplotlib in a notebook context, right?).

```suggestion
    - matplotlib-base
```

**Comment by jaimergp** [comment]
Yep, that's totally ok. Commit my suggestion and I will merge!

The install instructions cannot be updated on anaconda.org or the automatically generated README, but you can provide those in your repository.

**Comment by t-kimber** [comment]
Perfect, thanks a lot!!

**Pull request merged**